forge-gui/res/cardsfolder/upcoming/majestic_metamorphosis.txt

```diff
@@ -0,0 +1,8 @@
+Name:Majestic Metamorphosis
+ManaCost:2 U
+Types:Instant
+A:SP$ Animate | ValidTgts$ Artifact,Creature | TgtPrompt$ Select target artifact or creature | Power$ 4 | Toughness$ 4 | Types$ Angel,Artifact,Creature | AddKeyword$ Flying | SubAbility$ DBDraw | SpellDescription$ Until end of turn, target artifact or creature becomes a 4/4 Angel artifact creature and gains flying. Draw a card. | StackDescription$ Until end of turn, {c:Targeted} becomes a 4/4 Angel artifact creature and gains flying.
```

Comment:
Can you remove StackDescription and let me make this build right? I feel like the builder should be able to handle this